main.cpp

```diff
@@ -217,6 +242,17 @@ bool Bicycle::useBreak()
     return true;
 }
 
+void Bicycle::doesSomethingInteresting(int variableA)
+{
+    if (variableA >= 5)
+        std::cout << "Value is too big" << std::endl;
+    for(;;) {
+        if (variableA >= 5) break;
+        variableA +=1;
+        std::cout << "`variableA` --> " << variableA << std::endl;
+    }
```

Comment:
1) curly braces
2) space around `+=`
3) the code is easier to read/understand if you write it like this:
```
for( whatever )
{
    if( expr )
        break;
        
    statement;
    statement;
}
```
- [ ] revise